src/Components/CreatorStore.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import {
+  CampaignMarketContainer,
+  CampaignMarketSection,
+  CampaignMarketParticlesWrapper,
+  CreatorStyledInnerWrapper,
+  CreatorStyledLeftInner,
+  CreatorStyledRateBox,
+  ImageWrapper,
+  StyledText,
+  StyledTitle,
+  StyledSpan,
+  CreatorsStore,
+  RightInnerWrapper,
+  PopupWrapper,
+  RightWrapperContent,
+  StyledSpanText,
+  StyledParagraph,
+  GlobalStyles,
+} from './CreatorsStoreStyled';
+
+const CreatorStore = () => {
+  return (
+    <div>
+      <GlobalStyles />
+      <CampaignMarketSection>
+        <CampaignMarketContainer>
+          <CampaignMarketParticlesWrapper>
+            <img
+              src={'https://spreadd.io/assets/home/left-dot-patch.svg'}
+              alt="Logo"
+            />
+          </CampaignMarketParticlesWrapper>
+          <CreatorStyledInnerWrapper>
+            <CreatorStyledLeftInner>
+              <CreatorStyledRateBox>
+                <h5>100+</h5>
+                <p>Happy Customers</p>
+              </CreatorStyledRateBox>
+              <ImageWrapper>
+                <img
+                  src={'https://spreadd.io/assets/home/iphone.png'}
+                  alt="iPhone"
+                />
+              </ImageWrapper>
+              <PopupWrapper>
+                <h2>100+ curated products</h2>
+              </PopupWrapper>
+            </CreatorStyledLeftInner>
+            <RightInnerWrapper>
+              <RightWrapperContent>
+                <CreatorsStore>Creator Store</CreatorsStore>
+                <StyledText>
+                  <StyledTitle>Curate your</StyledTitle>
+                  <StyledTitle>products and turn</StyledTitle>
+                  <StyledTitle>your content shop-</StyledTitle>
+                  <StyledTitle>able</StyledTitle>
+                </StyledText>
+                <StyledParagraph>
+                  <StyledSpanText>
```

Comment:
same as above make that in one line code and set with width attribute @aeshapatel025